modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -7325,6 +7325,8 @@ feature.flag.LPS-171364.description=Users can move changes between publications.
 feature.flag.LPS-171364.title=Move Changes Between Publications
 feature.flag.LPS-174417.description=Site page and site template administrators will be able to see warnings when friendly URLs of pages already exist in the site template or the site created from the site template.
 feature.flag.LPS-174417.title=Site Template Friendly URL Conflict Warnings
+feature.flag.LPS-174455.description=Via Custom Objects OOTB API, users can add, get or update fields defined as attachment using Base64 format.
+feature.flag.LPS-174455.title=Base64 support for files management of Custom Objects
```

Comment:
Shoud we also set the feature flag as a release FF?

```
>> cat portal-impl/src/portal.properties | grep .type=release
    feature.flag.COMMERCE-9599.type=release
    feature.flag.COMMERCE-11922.type=release
    feature.flag.COMMERCE-12170.type=release
    feature.flag.COMMERCE-12192.type=release
    feature.flag.COMMERCE-12754.type=release
    feature.flag.LPD-10793.type=release
    feature.flag.LPD-11218.type=release
    feature.flag.LPD-15804.type=release
    feature.flag.LPS-114700.type=release
    feature.flag.LPS-161033.type=release
    feature.flag.LPS-164948.type=release
    feature.flag.LPS-165481.type=release
    feature.flag.LPS-182184.type=release
    feature.flag.LPS-186870.type=release
    feature.flag.LPS-187854.type=release
    feature.flag.LPS-188058.type=release
    feature.flag.LPS-188898.type=release
    feature.flag.LPS-193884.type=release
    feature.flag.LPS-196724.type=release
```

Reply:
fixed, thanks for noticing! 
```
cat portal-impl/src/portal.properties | grep .type=release
    feature.flag.COMMERCE-9599.type=release
    feature.flag.COMMERCE-11922.type=release
    feature.flag.COMMERCE-12170.type=release
    feature.flag.COMMERCE-12192.type=release
    feature.flag.COMMERCE-12754.type=release
    feature.flag.LPD-10793.type=release
    feature.flag.LPD-11218.type=release
    feature.flag.LPD-15804.type=release
    feature.flag.LPS-114700.type=release
    feature.flag.LPS-161033.type=release
    feature.flag.LPS-164948.type=release
    feature.flag.LPS-165481.type=release
    feature.flag.LPS-174455.type=release
    feature.flag.LPS-182184.type=release
    feature.flag.LPS-186870.type=release
    feature.flag.LPS-187854.type=release
    feature.flag.LPS-188058.type=release
    feature.flag.LPS-188898.type=release
    feature.flag.LPS-193884.type=release
    feature.flag.LPS-196724.type=release
```